Makefile

```
@@ -38,240 +33,223 @@ HEAD_VERSION = latest
-.PHONY: fix-backend
-fix-backend: fix-license fix-go fix-codegen
+.PHONY: serve-https
+serve-https: $(PRE)
+	@$(MAKE) --no-print-directory -C $(MODULES_DIRECTORY) serve-https
 
-.PHONY: check-frontend
-check-frontend: check-i18n check-license check-html check-scss check-ts
+.PHONY: run
+run: build-docker
+	docker run --rm -p 8001:8001 --net dashboard --name dashboard-web -d dashboard-web --locale-config /public/locale_conf.json --auto-generate-certificates
+	docker run --rm -p 9000:9000 -v $(KUBECONFIG):/config --net dashboard --name dashboard-api -d dashboard-api --kubeconfig config --auto-generate-certificates
+	docker run --rm -p 443:443 --net dashboard --name gateway -d gateway
 
-.PHONY: fix-frontend
-fix-frontend: fix-i18n fix-license fix-html fix-scss fix-ts
+.PHONY: build-docker
+build-docker: --ensure-docker-network build
+	@docker build -f hack/gateway/Dockerfile -t gateway .
+	@docker build -f modules/api/Dockerfile --build-arg BUILDPLATFORM=linux/amd64 -t dashboard-api .
+	@docker build -f modules/web/Dockerfile --build-arg BUILDPLATFORM=linux/amd64 -t dashboard-web .
 
-.PHONY: check
-check: check-i18n check-license check-go check-codegen check-html check-scss check-ts
-
-.PHONY: fix
-fix: fix-i18n fix-license fix-go fix-codegen  fix-html fix-scss fix-ts
-
-.PHONY: start-cluster
-start-cluster:
-	./aio/scripts/start-cluster.sh
-
-.PHONY: stop-cluster
-stop-cluster:
-	./aio/scripts/stop-cluster.sh
-
-.PHONY: e2e
-e2e: start-cluster
-	npm run e2e
-	make stop-cluster
-
-.PHONY: e2e-headed
-e2e-headed: start-cluster
-	npm run e2e:headed
-	make stop-cluster
-
-.PHONY: docker-build-release
-docker-build-release: build-cross
-	for ARCH in $(ARCHITECTURES) ; do \
-		docker buildx build \
-			-t $(RELEASE_IMAGE)-$$ARCH:$(RELEASE_VERSION) \
-			-t $(RELEASE_IMAGE)-$$ARCH:latest \
-			--build-arg BUILDPLATFORM=linux/$$ARCH \
-			--platform linux/$$ARCH \
-			--push \
-			dist/$$ARCH ; \
-	done ; \
-
-.PHONY: docker-push-release
-docker-push-release: docker-build-release
-	docker manifest create --amend $(RELEASE_IMAGE):$(RELEASE_VERSION) $(RELEASE_IMAGE_NAMES) ; \
-  docker manifest create --amend $(RELEASE_IMAGE):latest $(RELEASE_IMAGE_NAMES_LATEST) ; \
-  docker manifest push $(RELEASE_IMAGE):$(RELEASE_VERSION) ; \
-  docker manifest push $(RELEASE_IMAGE):latest
-
-.PHONY: docker-build-head
-docker-build-head: build-cross
-	for ARCH in $(ARCHITECTURES) ; do \
-		docker buildx build \
-			-t $(HEAD_IMAGE)-$$ARCH:$(HEAD_VERSION) \
-			--build-arg BUILDPLATFORM=linux/$$ARCH \
-			--platform linux/$$ARCH \
-			--push \
-			dist/$$ARCH ; \
-	done ; \
-
-.PHONY: docker-push-head
-docker-push-head: docker-build-head
-	docker manifest create --amend $(HEAD_IMAGE):$(HEAD_VERSION) $(HEAD_IMAGE_NAMES)
-	docker manifest push $(HEAD_IMAGE):$(HEAD_VERSION) ; \
+.PHONY: build
+build:
+	@$(MAKE) --no-print-directory -C $(MODULES_DIRECTORY) build
+
+.PHONY: --ensure-tools
+--ensure-tools:
+	@$(MAKE) --no-print-directory -C $(MODULES_DIRECTORY)/tools install
```

```suggestion
	@$(MAKE) --no-print-directory -C $(MODULES_DIRECTORY)/tools install; \
	$(MAKE) --no-print-directory -C $(MODULES_DIRECTORY)/web ci
```

If we're installing all of the go tooling, we should do the same for NPM